decidim-elections/app/presenters/decidim/admin_log/election_presenter.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Decidim
+  module AdminLog
+    # This class holds the logic to present a `Decidim::Election`
+    # for the `AdminLog` log.
+    #
+    # Usage should be automatic and you shouldn't need to call this class
+    # directly, but here's an example:
+    #
+    #    action_log = Decidim::ActionLog.last
+    #    view_helpers # => this comes from the views
+    #    ElectionPresenter.new(action_log, view_helpers).present
+    class ElectionPresenter < Decidim::Log::BasePresenter
+      private
+
+      def diff_fields_mapping
+        {
+          name: :i18n,
+          published_at: :date,
+          weight: :integer
+        }
+      end
+
+      def i18n_labels_scope
+        "activemodel.attributes.election"
+      end
+
+      def action_string
+        case action
+        when "publish", "unpublish"
+          "decidim.admin_log.election.#{action}"
```

Comment:
Maybe the structure would be even easier if you would use something like: 
`decidim.elections.admin_log.election.#{action}` - because `decidim.admin_log` could be in any module, while adding `elections` makes it very obvious. What do you think? 
You can find an example [here](https://github.com/decidim/decidim/blob/develop/decidim-budgets/config/locales/en.yml#L60)

Reply:
Done!

Reply:
Well, I was getting a RSpec error that was delaying too much this PR and I got back to the locales as they were. In a next PR I will fix it if possible.